dev/com.ibm.ws.microprofile.faulttolerance.cdi.2.1.services/src/com/ibm/ws/micrprofile/faulttolerance21/cdi/services/AnnotationConfigFactoryComponent.java

```diff
@@ -0,0 +1,20 @@
+/*******************************************************************************
+ * Copyright (c) 2018 IBM Corporation and others.
```

Comment:
date

Reply:
Did this and the other files in the same package.